src/backend/joanie/core/api/admin.py

```diff
@@ -71,6 +186,34 @@ class CertificateDefinitionViewSet(viewsets.ModelViewSet):
     filter_backends = [django_filters.rest_framework.DjangoFilterBackend]
     filterset_class = filters.CertificateDefinitionAdminFilterSet
 
+    def delete(self, request, *args, **kwargs):
+        """
+        Bulk deletion of CertificateDefinitions
+        data = {"id": [<id_0>, <id_1>]}
+        """
+        id_to_delete = request.data.get("id", [])
+        elements_to_delete = self.queryset.filter(id__in=id_to_delete)
```

Comment:
In order to be more efficient, we could use `self.queryset.filter(id__in=id_to_delete).delete()` but I don't know what append when `id_to_delete` contains an unknown resource.

The `delete` method seems to return deleted elements so we should be able to compare `deleted_ids` with `ids_to_delete` to return which elements cannot be deleted.

Reply:
In that case we'll only return elements that cannot be deleted and not those that were deleted

Reply:
I'll change that to return only the ids for all objects, deleted or not